app/javascript/viewer/uv_manager.js

```diff
@@ -213,8 +213,8 @@ export default class UVManager {
     this.uvElement.width(windowWidth)
     this.uvElement.height(windowHeight - titleHeight - tabHeight)
     this.uvElement.children('div').height(windowHeight - titleHeight - tabHeight)
-    if (this.uv) { this.uv.resize() }
     this.waitForElementToDisplay('button.share', 1000, this.addViewerIcons.bind(this))
+    if (this.uv) { console.log("resize"); this.uv.resize() }
```

Comment:
Do we want to keep this console log?

Reply:
Good catch, @hackartisan ! I removed it and pushed the commit.